src/mod_mam_cache_user.erl

```diff
@@ -97,6 +97,7 @@ writer_child_spec() ->
      [?MODULE]}.
 
 start_server(_Host) ->
+    %% TODO make per host server
```

Comment:
Can we introduce it in this PR? It doesn't seem to be big change and its one "TODO" less :D